### PR TITLE
Switch to zero-downtime-push

### PR DESCRIPTION
**Pull request by aeijdenberg**
Note that we'll need to manually delete the old apps after we run this the first time...

### CODE REVIEW

File: bin/cideploy.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ main() {
       cf auth $CF_USER $CF_PASSWORD
       cf target -o $CF_ORG
       cf target -s $CF_SPACE
-      cf push $appname
+      cf zero-downtime-push cgadocs -f manifest.yml --show-app-log
```
Comment (gordcorp) [inline bin/cideploy.sh:53]: Do we need to specify the appname `cgadocs` given it's in the manifest?

Comment (aeijdenberg) [reply bin/cideploy.sh:53]: Yes we do, the plugin gates doing a normal push vs a zero-downtime-deployment based on whether the app name is specified or not.  Maybe we should fix that on their end, but it's how it works for now.

Comment (aeijdenberg) [reply bin/cideploy.sh:53]: Actually, maybe we can remove from the manifest.yml.  Why don't I try that...

**Review by gordcorp — APPROVED** [review]
👍  Let's give it a try (tomorrow)

**Pull request merged**